app/models/ext_management_system.rb

```diff
@@ -271,6 +274,7 @@ def connection_configurations=(options)
 
     delete_unused_connection_configurations(options)
   end
+  alias connection_definitions= connection_configurations=
```

Comment:
The intention behind this is lost on me. I'd like to see some tests for this model showing all the use cases for these two attributes